crates/re_space_view/src/view_property_ui.rs

```diff
@@ -164,38 +164,35 @@ fn view_property_component_ui(
             .show_hierarchical(ui, singleline_list_item_content)
     };
 
-    list_item_response = list_item_response.on_hover_ui(|ui| {
+    list_item_response.on_hover_ui(|ui| {
         let id = egui::Id::new((archetype_name, field.display_name));
         ui.markdown_ui(id, field.docstring_md);
     });
-
-    view_property_context_menu(
-        ctx.viewer_ctx,
-        &list_item_response,
-        blueprint_path,
-        component_name,
-        component_results,
-    );
 }
 
-fn view_property_context_menu(
+fn menu_more(
     ctx: &ViewerContext<'_>,
-    list_item_response: &egui::Response,
+    ui: &mut egui::Ui,
     blueprint_path: &re_log_types::EntityPath,
     component_name: ComponentName,
     component_results: &re_query::LatestAtComponentResults,
 ) {
-    list_item_response.context_menu(|ui| {
-        if ui.button("Reset to default blueprint.")
+    let resolver = ctx.blueprint_db().resolver();
+
+    let property_differs_from_default = component_results.raw(resolver, component_name)
+        != ctx.raw_latest_at_in_default_blueprint(blueprint_path, component_name);
+    ui.add_enabled_ui(property_differs_from_default, |ui| {
+    if ui.button("Reset to default blueprint.")
         .on_hover_text("Resets this property to the value in the default blueprint.\n
         If no default blueprint was set or it didn't set any value for this field, this is the same as resetting to empty.")
+        .on_disabled_hover_text("The property is already set to the same value it has in the default blueprint.")
         .clicked() {
             ctx.reset_blueprint_component_by_name(blueprint_path, component_name);
             ui.close_menu();
         }
+    });
 
-        let blueprint_db = ctx.blueprint_db();
-        ui.add_enabled_ui(!component_results.is_empty(blueprint_db.resolver()), |ui| {
+    ui.add_enabled_ui(!component_results.is_empty(resolver), |ui| {
             if ui.button("Reset to empty.")
```

Comment:
```suggestion
            if ui.button("Reset to empty")
```

Reply:
Also, I wonder if "Unset" would be a better name. "empty" is somewhat vague, mysterious even in this context.

Reply:
agreed, unset is better